### DISCUSSION TITLE
Is there a paste generation data button somewhere?

### DISCUSSION THREAD

**Discussion by debevecc** [anapnoe/stable-diffusion-webui-ux#214] [Q&A]
I found the button that pastes the generation data from an image but is there a button to paste the generation data at the bottom of an image page from civitai?

**Comment by anapnoe** [comment]
no civitai support sorry
those buttons are against the good practices ui-ux tries to follow
I have to do a lot of extra work to make them optimized and compatible feel free to support the project so I can spare some time to look into it
